.storybook/preview.js

```diff
@@ -41,7 +41,7 @@ addParameters({
       h2: SectionName,
       h3: Title,
       li: AnchorListItem,
-      a: LinkComponent,
+      a: Link,
       p: Paragraph,
       Tip,
```

Comment:
Let's add also Link here, this way we'd be able to write `<Link` in our mdx files without importing it explicitly from the `vibe-storybook-components`
```suggestion
      Tip,
      Link,
```